client/stylesheets/_modal.scss

```diff
@@ -1181,10 +1181,62 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Profile Active Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+.modProfileActiveBtn {
+	margin: 20px 0 !important;
+	color: #fff !important;
+}
 
+.modProfileActiveBtn button {
+	padding: 15px !important;
+	width: 75% !important;
+	color: #fff !important;
+	background-color: #f8504b !important;
+	border-radius: 4px;
+}
 
+.modProfileActiveBtn button:hover {
+	background-color: #f73c37 !important;
+}
+
+.modProfileActiveBtn:first-child {
+	margin-top: 40px !important;
+}
 
+.modActiveCreator {
+	color: #00bfff !important;
+}
+
+.modActiveDirWrap {
+	text-align: center;
+}
+
+.modActiveDirWrap h3 {
```

Comment:
Merge rule `.modActiveDirWrap h3` with rule on line 1213<br>Selector `modActiveDirWrap` should be written in lowercase with hyphens